packages/redis/src/RedisClient.ts

```diff
@@ -20,24 +20,25 @@ import {
 } from "@walmartlabs/cookie-cutter-core";
 import { Span, SpanContext, Tags, Tracer } from "opentracing";
 import { isString } from "util";
-import { IRedisOptions } from ".";
+import { IRedisOptions, IRedisClient, RedisStreamID } from ".";
 import { RedisProxy } from "./RedisProxy";
 
-enum RedisMetrics {
+export enum RedisMetrics {
```

Comment:
i think you can undo making these `export` once you remove the explicit span from the sink